### DISCUSSION TITLE
Forgot Password link from the Login screen is messing up the keyboard TAB-ing through fields

### DISCUSSION THREAD

**Discussion by tmcdos** [community/community#68760] [Accessibility]
### Select Topic Area

Accessibility

### Body

I am the type of user who relies on Tabbing through form fields by keyboard. In the near past I was able to sign into Github by typing my username (thankfully the field is auto-focused), then press TAB key, then type my password, then press ENTER key, then type my 2FA code.

But since the last few days the SignIn panel has been changed - the `Forgot password` link has been put/moved between the `Username` and `Password` form fields. And since none of the form fields has a `tabindex` attribute - when I press the TAB key in my usual signin routine, type my password and press ENTER, I am being faced with the `Forgot password` dialog.

This change breaks my habbits of fast keyboard typing. Please be kind and either 
- change the tab order of the `Forgot password` link on the Login screen
- or move it back after/below the `Password` field as it was until just 2-3 days ago
![shot-1](https://github.com/community/community/assets/19253115/320bffae-c6f7-4bf4-9326-0020f5e63784)

**Comment by theultramage** [comment]
Your feedback should be in the Accessibility area, not in General. It is less likely to get addressed.
Your feedback title should describe the issue you're having, instead of what you think should be done. What you're proposing might not be the correct solution or might be outright rejected, and if you don't describe your problem then there's nothing more to do.

For me, the change broke my autotype signin, which relies on the Tab key to switch from username to password input. The webpage is missing any 'taborder' attributes, so now the 'forgot password' link gets in the way of the tabbing order. I think the correct solution would be to properly define it.

**Reply by tmcdos** [reply]
I updated the description as per your recommendations. If you think I can further improve it - let me know.

**Comment by tmcdos** [comment]
It looks like the issue has been fixed. Hopefully for a long time ahead :) (this is the second time it happens)

**Reply by ghostinhershell** [reply]
Hey @tmcdos 👋🏾  just wanted to follow-up here and see if this issue is indeed resolved.

**Reply by tmcdos** [reply]
Yes, TAB-bing in the login form now properly moves the focus from Username to Password and then to the Forgotten Password link. The fix is working fine. Thank you very much!

**Comment by cehfisher** [comment]
Hi @tmcdos - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/68760#discussioncomment-7250470). Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)

**Reply by tmcdos** [reply]
@cehfisher I confirm, the fix works properly.
